docs/howto/custom-management-commands.txt

```diff
@@ -191,6 +191,17 @@ Attributes
 All attributes can be set in your derived class and can be used in
 :class:`BaseCommand`’s :ref:`subclasses<ref-basecommand-subclasses>`.
 
+.. attribute:: BaseCommand.suppressed_base_arguments
+
```

Comment:
`versionadded` is missing. Please add also release notes.

Reply:
Added it to 3.2.6 release file.